docs/source/contributing/releasing.adoc

```diff
@@ -90,7 +90,7 @@ For more information, see xref:../contributing/writing-docs.adoc#[Writing and Pu
 
 As part of the release process we also send a release announcement and edit the GitHub release page.
 
-For the latter we usually add a categorized list of closed issues as well as some release highlights (most often taken from the release announcement).
+For the later we usually add a categorized list of closed issues as well as some release highlights (most often taken from the release announcement).
```

Comment:
@anjannath , `latter` only, `later` is different meaning here.

Reply:
Yeah.. Fixing it! ;-P